docs/developer-guide/api-overview.md

```diff
@@ -184,45 +186,30 @@ Styx core also add snippets of information like sender IP address, and so on.
 
 ### Http Handler Interface
 
-Http Handler interface forms a basis for Styx admin interfaces and routing objects. 
-As opposed to `HttpInterceptor`s, which just pass the messages down the pipeline,
-the `HttpHandler` is meant to *consume* the HTTP request. It is a similarly simple
-interface:
+A `HttpHandler` interface is the basis for Styx admin interfaces and routing objects. 
+It *consumes* the HTTP requests and asynchronously responds with HTTP responses. 
+It has a simple interface:
 
 ```java
     public interface HttpHandler {
-        StyxObservable<HttpResponse> handle(HttpRequest request, HttpInterceptor.Context context);
+        Eventual<LiveHttpResponse> handle(LiveHttpRequest request, HttpInterceptor.Context context);
     }
 ```
 
-It asynchronously processes the request, and returns the response within a context of 
-`StyxObservable`. It is given the `HttpInterceptor.Context` as a second argument, so that it is able
-to access the request context properties.
-
-Notice the absence of `Chain`. Therefore it is not able to proceed the message any further.
+It accepts a live HTTP request as its first argument, and eventually responds with live HTTP reponse. 
```

Comment:
"live HTTP reponse" -> "a live HTTP response"

I will be using `X -> Y` as shorthand for `Please change X to Y`.